nlp_primitives/tests/test_number_of_unique_words.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+import pandas as pd
+
+from ..number_of_unique_words import NumberOfUniqueWords
+from .test_utils import PrimitiveT, find_applicable_primitives, valid_dfs
+
+
+class TestNumberOfUniqueWords(PrimitiveT):
+    primitive = NumberOfUniqueWords
+
+    def test_general(self):
+        x = pd.Series(
+            [
+                "test test test test",
+                "test TEST test TEST",
+                "and;subsequent;lines...",
+                "$0.99 alteryx@alteryx.com",
+            ]
+        )
+
+        expected = pd.Series([1, 2, 3, 2])
```

Comment:
Shouldn't the expected for `test TEST test TEST` be 1?

Reply:
Default behavior is for `case_insensitive` to be `False`

Reply:
Got it!